app/src/main/java/amazingme/activities/app/LoginActivity.java

```diff
@@ -50,8 +50,7 @@ public void onClick(View view) {
             public void onClick(View view) {
                 final String email = emailEditText.getText().toString();
                 final String password = passwordEditText.getText().toString();
-                goTo(EnumeratedActivity.CHILD_REGISTRATION);
-                //AmazingMeApplicationContext.restoreSession(email, password, LoginActivity.this);
+                goTo(EnumeratedActivity.GAME_SELECTION_PAGE);
```

Comment:
why did you change this? 

definitely needs to be put back. it's fine if it's for testing, but make sure you put it back.